### DISCUSSION TITLE
does Go allowed

### DISCUSSION THREAD

**Discussion by Alhiane** [geeksblabla/blanat#22] [Q&A] [answered]
im go developer but seems it's not mentioned in chalnge allowed langs !?

**Comment by YassineOsip** [comment]
I don't think so it's included as @aboullaite mentioned before.

**Comment by aouakki** [comment, marked as answer]
@Alhiane i've added support to evaluate Golang solutions, I think you can use it now, cc @aboullaite

**Reply by aboullaite** [reply]
Yes, thanks to you @aouakki :)

**Reply by azihassan** [reply]
Are PRs accepted in the runner ? To add support for more languages

**Reply by aboullaite** [reply]
@azihassan yes 🙌
